src/SFA.DAS.ProviderApprenticeshipsService.Application.UnitTests/Commands/SubmitCommitment/WhenApprovingAsSecondApprover.cs

```diff
@@ -83,7 +98,81 @@ public void Setup()
             arg.Email.Tokens["type"].Should().Be("approval");
-            arg.Email.Tokens["cohort_reference"].Should().Be("ABC123");
+            arg.Email.Tokens["cohort_reference"].Should().Be(CohortReference);
+        }
+
+        [Test]
+        public async Task AndTransferFundedShouldSendNotificationOfApprovalToEmployer()
+        {
+            const string senderName = "Sender";
+            const string providerName = "Provider";
+
+            const string template =
+                @"Dear ((first_name)),
+
+((provider_name)) has approved cohort ((cohort_reference)).
+
+What happens next?
+A transfer request has been sent to ((sender_name)) for approval. You will receive a notification once ((sender_name)) approves or rejects the request.
+
+To view cohort ((cohort_reference)) and its progress, follow the link below.
+https://manage-apprenticeships.service.gov.uk/commitments/accounts/((employer_hashed_account))/apprentices/cohorts
+
+This is an automated message. Please do not reply to this email.
+
+Kind regards,
+
+Apprenticeship service team";
+
+            string expectedEmailBody = $@"Dear {UserName},
+
+{providerName} has approved cohort {CohortReference}.
+
+What happens next?
+A transfer request has been sent to {senderName} for approval. You will receive a notification once {senderName} approves or rejects the request.
+
+To view cohort {CohortReference} and its progress, follow the link below.
+https://manage-apprenticeships.service.gov.uk/commitments/accounts/{HashedEmployerAccountId}/apprentices/cohorts
+
+This is an automated message. Please do not reply to this email.
+
+Kind regards,
+
+Apprenticeship service team";
+
+            _commitmentView.TransferSender = new TransferSender
+                { Name = senderName };
+            _commitmentView.ProviderName = providerName;
+
+            SendNotificationCommand arg = null;
+            _mockMediator.Setup(x => x.SendAsync(It.IsAny<SendNotificationCommand>()))
+                .ReturnsAsync(new Unit()).Callback<SendNotificationCommand>(x => arg = x);
+
+            await _handler.Handle(_validCommand);
+
+            _mockMediator.Verify(x => x.SendAsync(It.IsAny<SendNotificationCommand>()), Times.Once);
+
+            arg.Email.RecipientsAddress.Should().Be("EmployerTestEmail");
+            arg.Email.TemplateId.Should().Be("EmployerTransferPendingFinalApproval");
+            arg.Email.Tokens["first_name"].Should().Be(UserName);
+            arg.Email.Tokens["cohort_reference"].Should().Be(CohortReference);
+            arg.Email.Tokens["sender_name"].Should().Be(senderName);
+            arg.Email.Tokens["provider_name"].Should().Be(providerName);
+            arg.Email.Tokens["employer_hashed_account"].Should().Be(HashedEmployerAccountId);
+
+            var emailBody = PopulateTemplate(template, arg.Email.Tokens);
+            TestContext.WriteLine(emailBody);
+            Assert.AreEqual(expectedEmailBody, emailBody);
```

Comment:
As per comment on previous PR, aren't we just testing the output of a private test function?

Reply:
As per previous reply, we're testing the template is correctly formed and we're storing a copy of the template in our repo.